test/test_utils.py

```diff
@@ -530,6 +530,7 @@ def _transform_MultiMarginCriterion(self, input, target):
         return input, target.sub(1)
 
 
+@unittest.skipIf('SKIP_TEST_BOTTLENECK' in os.environ.keys())
```

Comment:
Skips like this should have a comment pointing to the relevant bug explaining why we need to skip the test.

Reply:
don't know why it needs to be skipped, this is existing behavior

Reply:
I dont remember the answer to this one either :-/